### PR TITLE
External partner role

### PR DESCRIPTION
**Pull request by yeryomenkom**
#### APK
[Download build #12241](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12241/artifact/build/artifact/wire-dev-PR1957-12241.apk)
[Download build #12242](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12242/artifact/build/artifact/wire-dev-PR1957-12242.apk)
[Download build #12243](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12243/artifact/build/artifact/wire-dev-PR1957-12243.apk)
[Download build #12255](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12255/artifact/build/artifact/wire-dev-PR1957-12255.apk)
[Download build #12257](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12257/artifact/build/artifact/wire-dev-PR1957-12257.apk)
[Download build #12258](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12258/artifact/build/artifact/wire-dev-PR1957-12258.apk)
[Download build #12263](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12263/artifact/build/artifact/wire-dev-PR1957-12263.apk)
[Download build #12264](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12264/artifact/build/artifact/wire-dev-PR1957-12264.apk)
[Download build #12265](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12265/artifact/build/artifact/wire-dev-PR1957-12265.apk)

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -12,7 +12,7 @@ repositories {
 }
 
 ext {
-    majorVersion = "3.25."
```
Comment (JPonte) [inline app/build.gradle]: Why?

File: app/build.gradle [hunk 2]
```diff
@@ -21,10 +21,10 @@ ext {
     playServicesVersion = '15.0.1'
     audioVersion = System.getenv("AUDIO_VERSION") ?: '1.209.0@aar'
     stethoVersion = '1.5.0'
-    zMessagingDevVersion = "138.2205"
+    zMessagingDevVersion = "138.666-PR"
     paging_version = "1.0.0"
     // Release version number must be like this X.0(.Y)
-    zMessagingReleaseVersion = System.getenv("ZMESSAGING_VERSION") ?: '138.0.509@aar'
```
Comment (JPonte) [inline app/build.gradle]: Revert this too.

File: app/src/main/scala/com/waz/zclient/participants/ParticipantsAdapter.scala [hunk 3]
```diff
@@ -42,10 +42,12 @@ import com.waz.zclient.ui.text.{GlyphTextView, TypefaceEditText, TypefaceTextVie
 import com.waz.zclient.utils.ContextUtils._
 import com.waz.zclient.utils.{RichView, ViewUtils}
 import com.waz.zclient.{Injectable, Injector, R}
+import com.waz.ZLog.ImplicitTag._
 
 import scala.concurrent.duration._
 import com.waz.content.UsersStorage
 
+//TODO Maybe it will be better to split this adapter in two? One for participants and another for options?
```
Comment (JPonte) [inline app/src/main/scala/com/waz/zclient/participants/ParticipantsAdapter.scala]: I agree.

File: app/build.gradle [hunk 4]
```diff
@@ -21,10 +21,10 @@ ext {
     playServicesVersion = '15.0.1'
     audioVersion = System.getenv("AUDIO_VERSION") ?: '1.209.0@aar'
     stethoVersion = '1.5.0'
-    zMessagingDevVersion = "138.2205"
```
Comment (JPonte) [inline app/build.gradle]: Since we are merging to master, maybe merge also SE to develop and point to it?

Comment (yeryomenkom) [reply app/build.gradle]: Ok.

File: app/build.gradle (hunk 1 above)
Comment (yeryomenkom) [reply app/build.gradle]: I have no idea) seems like an accident

**Comment by AnastasiiaWire** [comment]
build #12258
bug #1 
- Sign in as a Partner
- I open Search UI
-  I tap on user name found on Search page <Member1>
-  I tap conversation name from top toolbar

Add people button is shown, it's possible to add people and create a group conversation

**Comment by julesmummdry** [comment]
build #12258
bug #2

- sign in as partner
- go into group details

You should see the option to change the notification settings for this group as partner

**Pull request merged**